_posts/en/newsletters/2022-12-14-newsletter.md

```diff
@@ -0,0 +1,287 @@
+      proposal accomplishes this by using one or more timelocked
+      transactions that ensure more recent states can be confirmed more
+      quickly than outdated states.
+
+        A consequence of this, described by Law, is that any LN payment
+        ([HTLC][topic htlc]) that is routed through a channel in a
+        channel factory needs to provide enough time for the latest
+        state timelock to expire so the factory can be unilaterally
+        closed.  Worse, this applies each time a payment is forwarded
+        through a factory.  For example, if a payment is forwarded
+        through 10 factories each with a 1-day expiry, it's possible that
+        a payment could be [jammed][topic channel jamming attacks] by
+        accident or on purpose for 10 days (or longer, depending on
+        other HTLC settings).
+
+    - *All or nothing:* for factories to truly achieve their best
+      efficiencies, all of their channels also need to be cooperatively
+      closed in a single onchain transaction.  Cooperative closes aren't
+      possible if any of the original participants becomes
+      unresponsive---and the chance of a participant becoming
+      unresponsive approaches 100% as the number of participants
+      increases, limiting the maximum benefit factories can provide.
+
+        Law cites previous work in allowing factories to remain
+        operational even if one participant wants to leave or,
+        conversely, one participant becomes unresponsive, such as the
+        proposals for `OP_TAPLEAF_UPDATE_VERIFY` and `OP_EVICT` (see
+        Newsletters [#166][news166 tluv] and [#189][news189 evict]).
+
+    Three proposed protocols are presented by Law to address the
+    concerns.  All derive from a previous proposal by Law [posted][law
+    tp] in October for *tunable penalties*---the ability to separate the
+    management of the enforcement mechanism (penalties) from the
+    management of other funds.  That previous proposal has not yet
+    received any discussion on the Lightning-Dev mailing list.  As of
+    this writing, Law's new proposal has also not received any
+    discussion.  If the proposals are sound, they would have the
+    advantage over other proposals of not requiring any changes to
+    Bitcoin's consensus rules.
+
+- **Local jamming to prevent remote jamming:** Joost Jager
+  [posted][jager jam] to the Lightning-Dev mailing list a link and
+  explanation for his project, [CircuitBreaker][].  This program,
+  designed to be compatible with LND, enforces limits on the number of
+  pending payments ([HTLCs][topic htlc]) the local node will forward on
+  behalf of each of its peers.  For example, consider the worst case
+  HTLC jamming attack:
+
+    ![Illustration of two different jamming attacks](/img/posts/2020-12-ln-jamming-attacks.png)
+
+    With the current LN protocol, Alice is fundamentally limited to
+    concurrently forwarding a maximum of 483 pending HTLCs.  If she
```

Comment:
Perhaps link `483 pending HTCLs` to https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#rationale-7